app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+  before_action :load_user, except: %i(create new)
+
+  def show; end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t "global.success"
```

Comment:
save thành công xong k redirect đi đâu à e?

Reply:
Em định làm log in rồi cho redirect lại log in luôn nma để em update lại cho chạy vào show ạ